hotsos.sh

```diff
@@ -29,6 +29,7 @@ export DATA_ROOT
 export SHOW_CPU_PINNING_RESULTS=false
 export AGENT_ERROR_KEY_BY_TIME=false
 # Path to the end product that plugins can see along the way.
+export OUTPUT_FORMAT
```

Comment:
eek sorry forgot one last thing. the comment above this applies to the line below so your line needs to go above the comment.

Reply:
Oops, sorry, I should have noticed that.